metadata-ingestion/tests/integration/redshift-usage/test_redshift_usage.py

```diff
@@ -82,6 +82,54 @@ def test_redshift_usage_source(pytestconfig, tmp_path):
     )
 
 
+@freeze_time(FROZEN_TIME)
+def test_redshift_usage_filtering(pytestconfig, tmp_path):
+
+    test_resources_dir = pathlib.Path(
+        pytestconfig.rootpath / "tests/integration/redshift-usage"
+    )
+
+    with patch(
+        "datahub.ingestion.source.usage.redshift_usage.RedshiftUsageSource._get_redshift_history"
+    ) as mock_event_history:
+        access_events = load_access_events(test_resources_dir)
+        mock_event_history.return_value = access_events
+
+        # Run ingestion
+        pipeline = Pipeline.create(
+            {
+                "run_id": "test-redshift-usage",
+                "source": {
+                    "type": "redshift-usage",
+                    "config": {
+                        "host_port": "xxxxx",
+                        "database": "xxxxx",
+                        "username": "xxxxx",
+                        "password": "xxxxx",
+                        "email_domain": "acryl.io",
+                        "include_views": True,
+                        "include_tables": True,
+                        "schema_pattern": {
```

Comment:
why not include a `table_pattern` as well to this test?

Reply:
Good point, I will include this as well

Reply:
Added this test case